tests/integration/integration_test_runner.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+NC='\033[0m'
+
+DATA_DIR="data"
+SUCCESS_COUNT=0
+FAILURE_COUNT=0
+
+# free port for dummy API
+lsof -ti tcp:5000 | xargs kill &> /dev/null
+
+# start the dummy API
+printf "%s\n" "" "    [TEST] Starting dummy APIs" ""
+python mock_dockerhub_api.py &> /dev/null &
+echo "done"
+
+cd ../..
+
+# run updater CLI
+printf "%s\n" "" "    [TEST] Running updater - no PR to create" ""
+rm -f last-diff-result.log
+source tests/integration/env.test && ONLY_UPDATE_TAGS_HISTORY=true ./main.sh
+
+# update an image
+printf "%s\n" "" "    [INFO] Pushing weaver tag to DockerHub" ""
+printf "${BLUE}"
+curl -s -XPOST localhost:5000/pavics/weaver/1.13.2-worker
```

Comment:
The version `1.13.2` is repeated at several places, should be factored out to a var.

Reply:
No, since  in the integration test, multiple tag versions are being used for different images.
Factoring out weaver and finch tags would, in my opinion, not improve readability of the integration test.

Reply:
It's also to prevent error when updating the tests.  If let's say the string "1.13.2" is in 1 variable, updating this var will handle everything.  Whereas now have to think about the initial POST to the dummy dockerhub, then the same value in the assert block.

Reply:
Yes ok, I see your point. Will be part of the integration test refac.

Reply:
Sorry I just thought about something.

Should add another test that fail the bump action.  Could be either be because of typo in `config.json` file like wrong location of `default.env` or typo in the `bump_tag` or because `bump_tag` value has really been renamed or moved to another location in birdhouse-deploy repo.

Either way, the error message should clearly state the root cause like `bump_file` not found or `bump_tag` not found to help diagnostic.